internal/commands/render.go

```diff
@@ -75,11 +76,14 @@ func prepareCustomAction(actionName string, dockerCli command.Cli, appname strin
 	if err != nil {
 		return nil, nil, nil, err
 	}
-	bundle, ref, err := cnab.ResolveBundle(dockerCli, bundleStore, appname)
+	bundle, ref, err := cnab.GetBundle(dockerCli, bundleStore, appname)
 	if err != nil {
+		if strings.HasSuffix(appname, ".dockerapp") {
+			return nil, nil, nil, fmt.Errorf("%q looks like a docker App directory and App must be built first before rendering", appname)
```

Comment:
Do we really want to be this specific? If we do, then why not print out the command the user needs to run to build the app?

Reply:
Maybe we could do the same as in docker app image tag who throw this type of error: "could not tag %q: no such App image".  So here something like: "could not render %q: no such App image"?